assets/js/components/ExecutionResults/ResultsContainer.test.jsx

```diff
@@ -10,22 +10,32 @@ import ResultsContainer from './ResultsContainer';
 
 describe('ChecksResults ResultsContainer component', () => {
   it('should render the notification box', () => {
-    renderWithRouter(<ResultsContainer catalogError />);
+    renderWithRouter(<ResultsContainer error />);
 
     expect(screen.getByRole('button')).toHaveTextContent('Try again');
   });
 
+  it('should render the notification box with a composite error', () => {
```

Comment:
```suggestion
  it('should render the notification box with a both errors', () => {
```

Reply:
What about `with several errors` instead of `with a both errors`?

Reply:
Looks good!